src/components/modules/Profile/Profile.tsx

```diff
@@ -42,7 +68,20 @@ const NEW_USER_QUERY = gql`
   }
 `
 
-const Profile: React.FC<PropType> = () => {
+const Profile: React.FC<PropType> = ({ me }) => {
+  const [isEditProfile, setEditProfile] = useState(false)
+  const classes = useStyles()
+  const currentUserId = useMemo(() => _get(me, 'uid'), [me])
+  const { loading: queryLoading, error: queryError, data: queryData } = useQuery(USER_QUERY, {
```

Comment:
you can use `useQuery<MeQuery, MeQueryVariables>(` here to have the variables type checked and the data types automatically. You can import these from hasuraTypes.ts